2019/12/27/from-dev-to-pm.html.md

```diff
@@ -0,0 +1,233 @@
+small project on its own.
+
+Again, estimating a project is an art in itself, and is not part of this post.
+
+Normally, in all of the above other than #1, you will need to confer with developers on your team
+to supply the client with an accurate estimate of costs. Of course, in order to be an efficient
+PM, you first need to
+</pre>
+### 3. Know your $developers
+<pre>
+This should be pretty self explanatory, but obviously you are not going to put a front end
+developer on a code intensive back end project.
+Another obvious point is that we are talking about a situation in which you have the luxury of
+multiple developers to chose from. If that is not the case, then you will be faced with another
+task, that of serving as a "translator" between the client and the developer.
+
+Given that statement, there are nuances that need to be considered when matching developers with
+clients, or how to control communications.
+
+If the client is a larger company, try to get them to assign a champion for the project, someone
+that is preferably somewhat technical, has authority to make some level of changes and decisions,
+and that is appointed by a high ranking person in the company. This person should be the contact
+point, representing the client for all daily project decisions and questions. In larger projects,
+this person may oversea a team from the client, that will interface with your team of developers.
+Without this person, you should warn the client that undo and excessive delays due to
+communications WILL occur.
+If the client is a smaller company, try to get an owner or other heavily committed person to be
+your point of contact. That will automatically escalate your questions so that you do not have
+to wait around for answers. You won't want to wait for answers, and the owner will not want to
+pay you an hourly rate to wait for answers either!
+
+All this so far is great, if you are the point of communication for your company. In many cases it
+will prove to be best for this to happen. You can monitor all conversations, communications, and
+prevent misunderstandings and hopefully anticipate problems before they occur.
+
+But if you need to have your developers talk directly to the client's rep, you have some thinking
+to do. I cannot give particulars here for obvious reasons, but certain traits do not mix well:
+If your client is very opinionated about their project, try to work out directions and methods
+BEFORE allowing comms between devs and client. (Use Trello, Slack, etc to assign specific tasks for
+specific people) Try to track progress of predefined tasks rather than agile development.
+
+If your client is tech savvy, easy to get along with, and you can convince your devs of that,
+direct interaction can be extremely productive. But don't just assume this will work, be on the
+the first phone calls or Zoom meetings.. observe the dynamics... try to be a fly on the wall..
```

Comment:
s/Zoom meetings/video conferences/ since there are many video conferencing options, not just Zoom